gradle/interop-as-source-set-klib.gradle

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+project.ext.registerInteropAsSourceSetOutput = { interopName, sourceSet ->
+    afterEvaluate {
+        def currentHostKotlinTarget = kotlin.targets.matching { it.platformType.name == "native" }.find {
+            it.konanTarget == hostManager.Companion.getHost()
+        }
+
+        def interop = currentHostKotlinTarget.compilations["main"].cinterops[interopName]
+        def cinteropTask = tasks.named(interop.interopProcessingTaskName)
+        def cinteropKlib = cinteropTask.map { it.outputFile }
+        def fakeCinteropCompilation = kotlin.targets["metadata"].compilations[sourceSet.name]
+        def destination = fakeCinteropCompilation.compileKotlinTask.destinationDir
+
+        def tempDir = "$buildDir/tmp/${sourceSet.name}UnpackedInteropKlib"
+
+        def prepareKlibTaskProvider = tasks.register("prepare${sourceSet.name.capitalize()}InteropKlib", Sync) {
+            from(files(zipTree(cinteropKlib).matching {
+                exclude("targets/**", "default/targets/**")
+            }).builtBy(cinteropTask))
+
+            into(tempDir)
+
+            doLast {
+                def manifest140 = file("$tempDir/default/manifest")
```

Comment:
Why such numbers? (140, 1371)

Reply:
PS: oh, I see it must stand for 1.4.0, 1.3.71, right?
Maybe `1_4_0` and `1_3_71` then?